packages/terra-date-picker/src/react-datepicker/stylesheets/react_datepicker.module.scss

```diff
@@ -356,8 +346,9 @@
 
   .react-datepicker-today-button {
     background: $datepicker-background-color;
-    border: 0;
-    border-top: 1px solid $datepicker-border-color;
+    border: 2px dashed transparent;
+    border-top: 2px solid $datepicker-border-color;
```

Comment:
Was changing the `border-top` to 2px an intentional change?
![image](https://user-images.githubusercontent.com/50138162/78019494-6cab0c00-736d-11ea-8f57-728b234be486.png)

Because in doing so, the border for the month body becomes inconsistent, i.e. the border above the month body is 1px and the border below the month body is 2px.

Reply:
Yes this was an intentional change to fix the jump issue as suggested in this [comment](https://github.com/cerner/terra-framework/pull/994#issuecomment-606011322)

Reply:
In that case, we should consider changing the top border of the month body as well, to keep the styling consistent.

Reply:
@jeremyfuksa What are you thoughts on this. Should the weight on the divider between the days of week and the month match the one below?

Reply:
Let's make them consistent.

Reply:
Done d882aad